app/models/package_manager/go.rb

```diff
@@ -9,6 +9,10 @@ class Go < Base
     KNOWN_HOSTS = [
       "bitbucket.org",
       "github.com",
+      "gitlab.com",
+      "gitee.com",
+      "gopkg.in",
+      "k8s.io",
```

Comment:
these are some of the more popular Go hosts in Libraries, in the long-tail following github.com